src/telegram.js

```diff
@@ -41,6 +41,7 @@ class TelegramBot extends EventEmitter {
    * @param {Boolean|Object} [options.webHook=false] Set true to enable WebHook or set options
    * @param {String} [options.webHook.key] PEM private key to webHook server.
    * @param {String} [options.webHook.cert] PEM certificate (public) to webHook server.
+   * @param {Boolean} [options.onlyFirstMatch=false] false: try matching all regexps; true: stop after first match
```

Comment:
Your wording is fine, feel free to reword; it's my first pull request, so I don't know how to change it.

I purposefully put documenting in separate commit than implementation, so one could be easily changed without other, because I wasn't sure if my wording was the best.